Storage/Podfile

```diff
@@ -4,11 +4,13 @@ target 'FirebaseStorageUI' do
   use_frameworks!
 
   pod 'Firebase/Storage'
-  pod 'SDWebImage', '~> 4.0'
-
-  target 'FirebaseStorageUITests' do
```

Comment:
See: https://stackoverflow.com/questions/40480503/the-bundle-uitests-couldn-t-be-loaded-because-it-is-damaged-or-missing-necessary

Current this Podfile seems cause the `FirebaseStorageUITests` run failed because it does not copy the Pod frameworks into the Test runner application.